From 7ca3782ea9d17e66189d8c16cc67e9763805729f Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Fri, 9 Feb 2018 13:20:37 +0800
Subject: [PATCH 3351/5242] MLK-17574 gpu: imx: dpu: Fix typos for
 scaler_scale_mode_t

commit  a35b9783dc7f2d74f53f5423085edc214b5e55da from
https://source.codeaurora.org/external/imx/linux-imx.git

Fix some typos for enum entry names of scaler_scale_mode_t.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/imx/dpu/dpu-plane.c |    4 ++--
 include/video/dpu.h                 |    4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/gpu/drm/imx/dpu/dpu-plane.c b/drivers/gpu/drm/imx/dpu/dpu-plane.c
index b3717d3..9eb60aa 100644
--- a/drivers/gpu/drm/imx/dpu/dpu-plane.c
+++ b/drivers/gpu/drm/imx/dpu/dpu-plane.c
@@ -568,7 +568,7 @@ static void dpu_plane_atomic_update(struct drm_plane *plane,
 		vscaler_output_size(vs, state->crtc_h);
 		vscaler_field_mode(vs, SCALER_INPUT);
 		vscaler_filter_mode(vs, SCALER_LINEAR);
-		vscaler_scale_mode(vs, SCLAER_UPSCALE);
+		vscaler_scale_mode(vs, SCALER_UPSCALE);
 		vscaler_mode(vs, SCALER_ACTIVE);
 		vscaler_set_stream_id(vs, dplane->stream_id ?
 					DPU_PLANE_SRC_TO_DISP_STREAM1 :
@@ -593,7 +593,7 @@ static void dpu_plane_atomic_update(struct drm_plane *plane,
 		hscaler_setup1(hs, src_w, state->crtc_w);
 		hscaler_output_size(hs, state->crtc_w);
 		hscaler_filter_mode(hs, SCALER_LINEAR);
-		hscaler_scale_mode(hs, SCLAER_UPSCALE);
+		hscaler_scale_mode(hs, SCALER_UPSCALE);
 		hscaler_mode(hs, SCALER_ACTIVE);
 		hscaler_set_stream_id(hs, dplane->stream_id ?
 					DPU_PLANE_SRC_TO_DISP_STREAM1 :
diff --git a/include/video/dpu.h b/include/video/dpu.h
index f28b2f2..8e97c66 100644
--- a/include/video/dpu.h
+++ b/include/video/dpu.h
@@ -383,8 +383,8 @@ enum dpu_irq {
 } scaler_filter_mode_t;
 
 typedef enum {
-	SCLAER_DOWNSCALE = 0x0,
-	SCLAER_UPSCALE = 0x10,
+	SCALER_DOWNSCALE = 0x0,
+	SCALER_UPSCALE = 0x10,
 } scaler_scale_mode_t;
 
 typedef enum {
-- 
1.7.9.5

